### DISCUSSION TITLE
customer feedback extension

### DISCUSSION THREAD

**Discussion by RonWalker22** [chatwoot/chatwoot#2083] [Show and tell]
I wanted to share my fork(early version). It's an extension to chatwoot to track, debate, and evaluate customer feedback. 

repo: https://github.com/ronwalker22/chatwoot-plus-feedback

2-minute demo:  https://www.youtube.com/watch?v=YpnfaD1Vp5s

chatwoot focuses on customer support. And although you can already manage customer feedback with labels and private messages, a dedicated system allows you to build powerful feedback-specific features. 

### Features

* Collecting feedback
  * At the moment, collecting feedback from your customers is accomplished by the existing channels provided by chatwoot. At some point down the road, I plan on adding modifications to some of these channels to enhance feedback collection. 

* Managing feedback
  * Track
    * Create feedback items for suggestions you wish to track
  * Debate
    * When creating feedback items you frame the feedback as a problem with potential solutions.
    * Agents can propose alternative solutions and provide pros and cons for each solution  
  * Evaluation
    * Agents can vote on feedback items and their solutions. 
    * After review by the entire team, admins can select the best solution(s) to be implemented to solve the problem.
      * admins can optionally add any final thoughts on the "judgment tab."

**Comment by VSinghhh** [comment]
How come a PR request wasn't made for merging this? - its a fantastic idea and implementation @RonWalker22
